### ISSUE TITLE
"Bridge" mode improvements / additions

### ISSUE THREAD

**Issue by edfletcher** [discord-relay-chat/drc#35]
As more people are using DRC as a bridge between Discord and IRC (a use case I did not initially design or build for but that DRC naturally does pretty well), I'd like to put focus on improving the rough edges of this use case and adding features that support it.

- [ ] WrEtCh/eradev suggested an excellent addition, but one which would require a fair bit of work: individual IRC connections for each Discord user. They even kindly shared [a repo](https://github.com/waveplate/discord-irc-relay) that does this, with `irc-framework` no less! We would do similar: instead of having the `config.irc.registered` structure, each Discord user would have a connection to the specified IRC server opened on their behalf, use their Discord nick as the IRC nick.

**Comment by edradev** [comment]
Yes, @edfletcher perfectly described. Ofcourse the separate connections to the specified IRC server will be opened in case of activity at the discord channel (user talks) and after some inactivity lets say few hours the user will be disconected from irc (we don't want to have much inactive/idle users/connections)
